lib/cc/services/github_pull_requests.rb

```diff
@@ -104,6 +107,9 @@ def update_status(state, description)
         target_url:  @payload["details_url"],
         context:     "codeclimate"
       }
+      if state == "error"
+        params.delete(:target_url)
```

Comment:
Agreed that change would be better in finalizer.

> without adding in a conditional, it might set the target_url to nil when @payload["details_url"] gets left out.

I would test if this matters. If it does, I'd rather *that* conditional than the one you have.